src/style.css

```diff
@@ -1 +1,99 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header_style {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-content: center;
+}
+
+.logo_image {
+  display: flex;
+  flex-wrap: wrap;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  align-items: center;
+  align-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.menu-item {
+  display: flex;
+  align-items: center;
+  align-self: center;
+}
+
+.menu-item:nth-child(1) {
+  margin-right: 10px;
+}
+
+.menu-item:last-child {
+  margin-left: 10px;
+}
+
+.menu-item:nth-child(n+2):not(:last-child) {
+  margin: 0 10px;
+}
+
+.menu_link {
+  display: block;
+  position: relative;
+  padding: 23px 0;
+  color: rgba(0, 0, 0, 1);
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.menu_link:link {
+  text-decoration: none;
+}
+
+.menu_link:visited {
+  text-decoration: none;
+}
+
+.menu_link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.menu_link:active {
+  text-decoration: none;
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
```

Comment:
Move this color to a variable. And use rgb or hex. You don't need to use rgba here